Sources/HTTPTypes/HTTPField.swift

```diff
@@ -181,8 +195,43 @@ extension HTTPField: CustomPlaygroundDisplayConvertible {
     }
 }
 
+extension HTTPField: Codable {
+    enum CodingKeys: String, CodingKey {
+        case name
+        case value
+        case indexingStrategy
+    }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(self.name, forKey: .name)
+        try container.encode(self.rawValue._storage, forKey: .value)
+        if self.indexingStrategy != .automatic {
+            try container.encode(self.indexingStrategy.rawValue, forKey: .indexingStrategy)
+        }
+    }
+
+    private enum DecodingError: Error {
+        case invalidValue(String)
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let name = try container.decode(Name.self, forKey: .name)
+        let value = try container.decode(String.self, forKey: .value)
+        guard Self.isValidValue(value) else {
+            throw DecodingError.invalidValue(value)
+        }
+        self.init(name: name, uncheckedValue: ISOLatin1String(unchecked: value))
```

Comment:
Just for my understanding. We can safely round trip through the `ISOLatin1String` right? There are no values that might get lost.

Reply:
Yes, ISOLatin1String can represent any byte value.